### PR TITLE
Fix "classs" typos

### PR DESCRIPTION
**Pull request by benasocj**

### CODE REVIEW

File: pkg/analyzer/test/src/dart/analysis/driver_test.dart [hunk 1]
```diff
@@ -639,7 +639,7 @@ part 'foo.dart';
     provider.newFile(a, 'class A {}');
     provider.newFile(b, "import 'a.dart'; A a;");
     provider.newFile(c, "import 'a.dart'; var a = new A();");
-    provider.newFile(d, "classs A{} A a;");
```
Comment (ben4ever) [inline pkg/analyzer/test/src/dart/analysis/driver_test.dart]: Don't know if this change will have any side effects.

**Comment by mit-mit** [comment]
@a-siva can you take a look please?

**Comment by floitschG** [comment]
I think this is not for Siva but for @bwilkerson

**Comment by a-siva** [comment]
Make sense I will reassign the bug to @bwilkerson as it seems to be in the analysis driver test.

**Review by bwilkerson — APPROVED** [review]

**Comment by mit-mit** [comment]
Thanks for the contribution, @ben4ever !

**Pull request merged**